iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/ActivePowerControlAdderImpl.java

```diff
@@ -47,4 +47,18 @@ public ActivePowerControlAdder<I> withParticipationFactor(double participationFa
         return this;
     }
 
+    @Override
+    public boolean getParticipate() {
+        return this.participate;
+    }
+
+    @Override
+    public double getDroop() {
+        return this.droop;
+    }
+
+    @Override
+    public double getParticipationFactor() {
+        return this.participationFactor;
+    }
```

Comment:
to remove, you did this because of the incorrect g.getExtension(ActivePowerControlAdder.class) above